shuffle/move/examples/e2e/message.test.ts

```diff
@@ -27,13 +27,56 @@ Deno.test("Ability to set message", async () => {
 });
 
 Deno.test("Ability to set NFTs", async () => {
+  // Initialize nft_collection resource for both sender and receiver
+  let senderInitializeTxn = await main.initializeNFTScriptFunction(
+    context.defaultUserContext,
+  );
+  senderInitializeTxn = await devapi.waitForTransactionCompletion(
+    senderInitializeTxn.hash,
+  );
+  const secondUserContext = context.UserContext.fromEnv("test");
+  let receiverInitializeTxn = await main.initializeNFTScriptFunction(
+    secondUserContext,
+  );
+  receiverInitializeTxn = await devapi.waitForTransactionCompletion(
+    receiverInitializeTxn.hash,
+  );
+  assert(senderInitializeTxn.success);
+  assert(receiverInitializeTxn.success);
```

Comment:
In a fast follow, let's cut down on the LOCs to be more readable and only focus on what we want to share. These are all just sanity tests to ensure proper setup and can now be cut. cc @sunmilee